src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Certain machine started its work");

```

Comment:
[Bad example of a message. It should be unique for every single implementation. ](https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/oop#write-informative-messages-in-methods)
```suggestion
        System.out.println("Bulldozer started its work");

```
Fix in all places